### PR TITLE
 Provide 'Reset template' button for subscribe pages

### PR DESCRIPTION
**Pull request by xh3n1**
https://mantis.phplist.org/view.php?id=19267

### CODE REVIEW

**Comment by xh3n1** [comment]
This PR on trevelin theme needs to merged for the resettemplate icon : https://github.com/phpList/phplist-ui-bootlist/pull/60

screen shot 2018-06-28 at 14 24 20

File: public_html/lists/admin/spage.php [hunk 1]
```diff
@@ -42,6 +42,37 @@
...
+if (isset($_REQUEST['reset'])) {
+    $reset = sprintf('%d', $_REQUEST['reset']);
+} else {
+    $reset = 0;
+}
+
+
+if ($reset) {
+    $valuesToUpdate =  array(
+        'header' => 'pageheader',
```
Comment (xh3n1) [inline public_html/lists/admin/spage.php]: @samtuke These are the values that will be reset, please let me know if there is something else that needs to be updated (or not).

**Comment by samtuke** [comment]
@x3hn1 Thanks!

@suelap What do you think about only resetting the subscribe page header and footer, so that the reset only affects the theme and not the page text content or transactional messages?

**Comment by suelaP** [comment]
@samtuke yes I think it would be better to use this option to reset only header and footer, but we should also do something similar with updating transactional messages, in a seperate issue.

**Comment by samtuke** [comment]
@suelaP @xh3n1 Should the wording of the button alt text be changed to reflect this? E.g. 'Reset styling to default (header and footer)'?

**Comment by suelaP** [comment]
I think "Reset styling to default" it's enough of a hint. The message then provides a better description. @xh3n1 @samtuke

**Comment by xh3n1** [comment]
@samtuke @suelaP done.

**Pull request merged**